frontend-react/cypress/e2e/specs/05-submissions-page.cy.js

```diff
@@ -0,0 +1,37 @@
+import submissionsPage from "../../support/pages/Submissions";
+
+describe("Submissions Page", () => {
+    let submissionsResponse;
+
+    beforeEach(() => {
+        cy.login();
+        // TODO: parameterize the organization
+        cy.setOrganization("ignore");
+        submissionsPage.go();
+
+        // TODO: parameterize the format
+        cy.fetchSubmissions("ignore", "CSV").then((response) => {
+            submissionsResponse = response;
+        });
+    });
+
+    it("renders the correct heading", () => {
+        submissionsPage
+            .getElements()
+            .heading.should("contain", "Submission History");
+    });
+
+    // TODO: kind of a dumb test -- could check for pagination options?
+    // same dumb assertion in daily-data-page
+    it("renders a table with up to ten rows", () => {
+        submissionsPage.paginatedTable
+            .getElements()
+            .tableRows.should(
+                "have.length",
+                submissionsPage.paginatedTable.getInitiallyDisplayedRows(
+                    submissionsResponse.body.length,
+                    10
```

Comment:
It looks like this will be an issue across both Daily Data and Submission History -- at least semantically -- since it'll always expect 10 exactly rather than "up to ten"

Reply:
No, the table expects at most ten here: https://github.com/CDCgov/prime-reportstream/pull/7913/files#diff-7b3c7772796e19f538dd4c3cd0c6c10045f9133750d88bd58b19afc1d433f2fcR20

It's configurable on a per-PaginatedTable instance, but I added 10 as a default since that seems like a reasonable default 🤔

Reply:
I don't think we should assume/assert that any table has any minimum number of displayed rows since these (in theory) are running in 3 different environments with varying levels of "realism" in the availability of data; least-to-most normal being dev -> staging -> prod. I'm still receiving the Daily Data table bug after pulling latest, for example, despite near-certainty that we'd have 10 rows for the `ignore` data in Staging, and full certainty that we'd have 0 in Prod since `ignore` is not a present org in Prod. Maybe the test could be reframed as "no _more_ than 10 rows" to assert `pageSize` rather than asserting the existence of `x` rows of data or more.

Reply:
Hmmm maybe I'm misunderstanding the issue you're raising, but this should be what the spec is doing:

- Make an API call to fetch the records
- Render the page
- Check that there are up to n rows (maximum of ten) rendered based on the number of records from the API response

This is the same for [the Daily Data page](https://github.com/CDCgov/prime-reportstream/pull/7913/files#diff-0fdf2ea72678601f9324e8d69c149efab94342d60469efcd4ca98dc8b2c4a001R12-R16) and [the Submissions page](https://github.com/CDCgov/prime-reportstream/pull/7913/files#diff-2f8cc86cc0f8e62b7eff73706c3e32f34a1735c23aa6c06c253ea32048118ffdR13-R17).  The idea behind this was to accommodate a variable database state since that's not for sure.  If there are 0 records returned, it should assert that there are 0 rows rendered; if there are 10+ records returned, it should assert that there are 10 rows rendered.  The biggest assumption here is on the `ignore` organization, which we can assume for local development.

I'll look into why you're running into the bug, though!

Reply:
Ah!  The bug was caused by my dumb mistake: [the `fetchDeliveries` command](https://github.com/CDCgov/prime-reportstream/pull/7913/files#diff-178021bf246c859cc0f0eac4635d9a106811692534564dea6cd694aa7b387cc9R81) was being mapped to the `fetchReceivers` function.  Should be good to go now -- sorry about that!